lib/core/shared/sdk/embeddedSdk.ts

```diff
@@ -134,6 +161,24 @@ export class EmbeddedSDK extends Kuzzle {
         : options.propagate;
     }
 
+    if (deprecatedForbiddenEmbeddedActions[request.controller] !== undefined
+      && deprecatedForbiddenEmbeddedActions[request.controller].has(request.action)
+    ) {
+      throw kerror.get(
```

Comment:
This errors should tell the user that those methods cannot be used with the EmbeddedSDK